pkg/version.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func GetCurrentVersion() string {
-	return "0.94.11" // ci-version-check
+	return "0.94.12" // ci-version-check
```

Comment:
I think you can remove, because it's already 0.94.12 (just rebase), as for the version bump, you can do it against main directly IMO